KinveyKit/KinveyKit/Source/KCSNetworkResponse.m

```diff
@@ -129,7 +129,7 @@ - (id) jsonResponseValue:(NSError**) anError
                                                                         options:NSJSONReadingMutableContainers
                                                                           error:&error];
     NSObject* jsonObj = nil;
-    if (![jsonResponse isKindOfClass:[NSDictionary class]]) {
+    if (!([jsonResponse isKindOfClass:[NSDictionary class]] || [jsonResponse isKindOfClass:[NSArray class]])) {
```

Comment:
@heyzooi the variable `jsonResponse` is of `NSMutableDictionary`. In what case will it be an array?